enrollment-server/src/main/java/com/wultra/app/enrollmentserver/database/DocumentResultRepository.java

```diff
@@ -30,4 +33,13 @@
 @Repository
 public interface DocumentResultRepository extends CrudRepository<DocumentResultEntity, Long> {
 
+    /**
+     * @return All not finished document uploads (in progress status and no extracted data filled)
+     */
+    @Query("select doc from DocumentResultEntity doc where" +
```

Comment:
Perhaps we should capitalize SQL commands like in other queries?

Reply:
fixed